### PR TITLE
Fix flow of nopia build props

### PR DESCRIPTION
**Pull request by tmat**
By default PackageReference does not flow build targets.

### CODE REVIEW

File: src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj [hunk 1]
```diff
@@ -177,9 +177,11 @@
     <PackageReference Include="VsWebsite.Interop" Version="$(VsWebsiteInteropVersion)" />
     <PackageReference Include="NuGet.VisualStudio" Version="$(NuGetVisualStudioVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
-    <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="$(MicrosoftVisualStudioSDKEmbedInteropTypesVersion)" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowVersion)" />
-    <PackageReference Include="Microsoft.VisualStudio.StaticReviews.Embeddable" Version="$(MicrosoftVisualStudioStaticReviewsEmbeddableVersion)" />
+
+    <!-- By default build assets that define embedded interop types do not flow. Set PrivateAssets to none to make them flow. -->
+    <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="$(MicrosoftVisualStudioSDKEmbedInteropTypesVersion)" PrivateAssets="none" />
+    <PackageReference Include="Microsoft.VisualStudio.StaticReviews.Embeddable" Version="$(MicrosoftVisualStudioStaticReviewsEmbeddableVersion)" PrivateAssets="none" />
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj:184]: There's build assets in this too? I thought we had a special rule in our VisualStudio.props that was also embedding this?

Comment (tmat) [reply src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj:184]: Yes, there is one. We can probably remove ours.

**Comment by tmat** [comment]
Are integration tests broken?
@sharwell

**Comment by genlu** [comment]
Good to merge?

**Comment by tmat** [comment]
Yes, thanks for merging.

**Pull request merged**